bin/tests/test_model/titanic_model.py

```diff
@@ -0,0 +1,33 @@
+import torch
+import torch.nn as nn
+
+class ModelTitanic(nn.Module):
+    def __init__(self, nb_neurons_intermediate_layer: int = 7, nb_intermediate_layers: int = 3, nb_classes: int = 2):
+        super(ModelTitanic, self).__init__()
+        self.input_layer = nn.Linear(7, nb_neurons_intermediate_layer)
+        self.intermediate = nn.modules.ModuleList([nn.Linear(nb_neurons_intermediate_layer, nb_neurons_intermediate_layer) for _ in range(nb_intermediate_layers)])
+        self.output_layer = nn.Linear(nb_neurons_intermediate_layer, nb_classes)
+        self.relu = nn.ReLU()
+        self.softmax = nn.Softmax(dim=1)
+
+    def forward(self, pclass, sex, age, sibsp, parch, fare, embarked):
+        # print all inputs
+        x = torch.stack((pclass, sex, age, sibsp, parch, fare, embarked), dim=1).float()
+        x = self.relu(self.input_layer(x))
+        for layer in self.intermediate:
+            x = self.relu(layer(x))
+        x = self.softmax(self.output_layer(x))
+        return x
+    
+    def compute_loss(self, loss_fn, output, survived):
+        return loss_fn(output, survived)
+    
+    def epoch(self, x, y, loss_fn, optimizer):
```

Comment:
also loss_fn should be a dict, isn't it? 
So that for each target y, it computes certain loss, according to the config

Reply:
loss_fn is a argument, name should match the losses defined in the config with the losses defined in the "epoch" method, thus, if the user has to apply various loss_fn, user could define multiple losses with the appropriate names in the epoch definition.